components/link/link.js

```diff
@@ -164,7 +164,7 @@ class Link extends LocalizeCoreElement(FocusMixin(LitElement)) {
 			'd2l-link-small': this.small,
 			'truncate': this.lines > 0
 		};
-		const styles = (this.lines > 0) ? { '-webkit-line-clamp': this.lines } : {};
+		const styles = { whiteSpace: 'nowrap', webkitLineClamp: this.lines || null };
```

Comment:
Since these get put inline, I think typically we'd only put dynamic stuff in these style maps? vs. just putting it up in the CSS?

Reply:
lol, yes, of course.